spec/test/fixtures/server_error/tests/server-error.agent.nut

```diff
@@ -0,0 +1,7 @@
+
+class AgentTestCase extends ImpTestCase {
```

Comment:
I've seen this code at least 4 times. Can you please fix this?

One option is to use `@include`